nixos/lib/test-driver/Machine.pm

```diff
@@ -43,7 +43,7 @@ sub new {
             if defined $args->{bios};
         $startCommand .= $args->{qemuFlags} || "";
     } else {
-        $startCommand = Cwd::abs_path $startCommand;
+        #$startCommand = Cwd::abs_path $startCommand;
```

Comment:
@edolstra any suggestions? I'd really like to merge this in soon so I can move onto the more interesting PR :smile: